github/repos_pages.go

```diff
@@ -62,6 +70,51 @@ func (s *RepositoriesService) EnablePages(ctx context.Context, owner, repo strin
 	}
 
 	return enable, resp, nil
+
+}
+
+type RemoveCName struct {
```

Comment:
Please change the name of this struct to `removeCName` so that it is not exported for external use.